velox/functions/prestosql/tests/BitwiseTest.cpp

```diff
@@ -359,47 +360,45 @@ TEST_F(BitwiseTest, rightShift) {
 }
 
 TEST_F(BitwiseTest, logicalShiftRight) {
-  const auto bitwiseLogicalShiftRight = [&](std::optional<int64_t> number,
-                                            std::optional<int64_t> shift,
-                                            std::optional<int64_t> bits) {
+  const auto evalFunc = [&](std::optional<int64_t> number,
+                            std::optional<int64_t> shift,
+                            std::optional<int64_t> bits) {
     return evaluateOnce<int64_t>(
         "bitwise_logical_shift_right(c0, c1, c2)", number, shift, bits);
   };
 
-  EXPECT_EQ(bitwiseLogicalShiftRight(1, 1, 64), 0);
-  EXPECT_EQ(bitwiseLogicalShiftRight(-1, 1, 2), 1);
-  EXPECT_EQ(bitwiseLogicalShiftRight(-1, 32, 32), 0);
-  EXPECT_EQ(bitwiseLogicalShiftRight(-1, 30, 32), 3);
-  EXPECT_EQ(bitwiseLogicalShiftRight(kMin64, 10, 32), 0);
-  EXPECT_EQ(bitwiseLogicalShiftRight(kMin64, kMax64, 64), -1);
-  EXPECT_EQ(bitwiseLogicalShiftRight(kMax64, kMin64, 64), kMax64);
-
-  VELOX_ASSERT_THROW(
-      bitwiseLogicalShiftRight(3, -1, 3), "Shift must be positive");
-  VELOX_ASSERT_THROW(
-      bitwiseLogicalShiftRight(3, 1, 1), "Bits must be between 2 and 64");
+  EXPECT_EQ(evalFunc(1, 1, 64), 0);
+  EXPECT_EQ(evalFunc(-1, 1, 2), 1);
+  EXPECT_EQ(evalFunc(-1, 32, 32), 0);
+  EXPECT_EQ(evalFunc(-1, 30, 32), 3);
+  EXPECT_EQ(evalFunc(kMin64, 10, 32), 0);
+  EXPECT_EQ(evalFunc(kMin64, kMax64, 64), -1);
+  EXPECT_EQ(evalFunc(kMax64, kMin64, 64), kMax64);
+  EXPECT_EQ(evalFunc(7, 0, 64), 7);
+
+  VELOX_ASSERT_THROW(evalFunc(3, -1, 3), "Shift must be non-negative");
+  VELOX_ASSERT_THROW(evalFunc(3, 1, 1), "Bits must be between 2 and 64");
 }
 
 TEST_F(BitwiseTest, shiftLeft) {
```

Comment:
should this be `logicalShiftRight` to be consistent?

Reply:
@kewang1024 
I don't have a strong opinion, I just saw this discrepancy and renamed the local vars to something neutral, so it can be easily copy-pasted w/o breaking any logic (code or simply naming).
The variables that have distinct names are the member of the test class, while these are locals.